modules/gapi/src/api/kernels_imgproc.cpp

```diff
@@ -97,6 +97,22 @@ GMat Canny(const GMat& src, double thr1, double thr2, int apertureSize, bool l2g
     return imgproc::GCanny::on(src, thr1, thr2, apertureSize, l2gradient);
 }
 
+std::tuple<GArray<Point2f>, GArray<uchar>, GArray<float>> calcOpticalFlowPyrLK(const GMat& prevImg,
+    const GMat& nextImg, const GArray<Point2f>& prevPts, const GArray<Point2f>& predPts, const Size& winSize,
+    int maxLevel, const TermCriteria& criteria, int flags, double minEigThresh)
```

Comment:
Please replace return type and all GArray<*> types with the types you defined in modules/gapi/include/opencv2/gapi/imgproc.hpp.